app/controllers/api/v2/smart_class_parameters_controller.rb

```diff
@@ -36,6 +36,7 @@ def show
         param :override, :bool, :desc => N_("Whether the smart class parameter value is managed by Foreman")
         param :description, String, :desc => N_("Description of smart class")
         param :default_value, String, :desc => N_("Value to use when there is no match")
+        param :hidden_value, :bool, :desc => N_("Value to display when the parameter should be hidden")
```

Comment:
Not really a "value to display", because it's a boolean that controls *if* the parameter should be hidden.  Perhaps: "Hide the parameter value [in the UI?] when enabled"